packages/test/end-to-end-tests/src/test/loaderTest.spec.ts

```diff
@@ -33,6 +35,26 @@ class TestSharedDataObject1 extends DataObject {
     public get _context() {
         return this.context;
     }
+
+    public async request(request: IRequest): Promise<IResponse> {
```

Comment:
Can you add comment that this method is used to return the query params for the query params test.